src/machine.h

```diff
@@ -288,6 +294,36 @@ class machine final {
     /// \brief Destructor.
     ~machine();
 
+    /// \brief Fill file descriptors to be polled by select() for all VirtIO devices.
+    /// \param fds Pointer to sets of read, write and except file descriptors to be updated.
+    /// \param timeout_us Maximum amount of time to wait in microseconds, this may be updated (always to lower values).
+    void poll_virtio_devices_before_select(select_fd_sets *fds, uint64_t *timeout_us);
```

Comment:
This function name confused. It seems like this function does poll the virtio devices before some unrelated call to select. I'd just say prepare_virtio_devices_for_select.

Reply:
Changed the names.